April Serenity updates (#589)
akeneo/pim-helpcenter: content/updates/04-2020/5-data-quality-new-improvements.md

# Data quality improvements

## Data Quality Insights on product models

> ::: meta-data type="Improvement" features="Quality, Data Quality Insights" available="5th of March" link-to-doc="../articles/articles/understand-data-quality.html"

Reviewer: Not the good date, it's written March but it's supposed to be in April

Reviewer: See with Flo, it's mid-April.
```suggestion
::: meta-data type="Improvement" features="Quality, Data Quality Insights" available="mid-April" link-to-doc="../articles/articles/understand-data-quality.html"
```

:::

> Following last month improvements on product variants, the data quality insights tab listing recommendations and grades is **now also available for product models**. You can now see the enrichment and consistency grading scores for all attributes and see at glance on which level the attribute is (product variant or model level).

Reviewer 2: ```suggestion
Following last month improvements on product variants, the data quality insights tab listing recommendations and grades is **now also available for product models**. You can now see the enrichment and consistency grading scores for all attributes and see at a glance on which level the attribute is (product variant or model level).
```

This improvement now enables you to easily monitor **your data quality and accuracy for all your products**.

![Data Quality Insights on variant products](../img/data-quality-variant-panel_2.png)

::: more

> [Understand data quality](../articles/articles/understand-data-quality.html)

Reviewer: ```suggestion
[What is data quality?](../articles/articles/understand-data-quality.html)
```

> :::

Reviewer: Delete this. There should be only one per md file.

## Spell-check: new languages supported!

> ::: meta-data type="Improvement" features="Quality, Data Quality Insights" available="30th of April"

Reviewer: ```suggestion
::: meta-data type="Improvement" features="Quality, Data Quality Insights" available="late April"
```

:::

Three new locales are now supported for the spell-check features: Norwegian Bokmål 🇳🇴, Norwegian Nynorsk 🇳🇴 and Brazilian Portuguese 🇧🇷.

> The spell-check feature now can help you **improve your product information and avoid mistakes for 11 locales**!

Reviewer 2: ```suggestion
The spell-check feature can now help you **improve your product information and avoid mistakes for 11 locales**!
```

> ## For single locale PIM

Reviewer: It's missing the meta-data here.

> Prior to this release, only localisable attributes could benefit from the spell-check feature released in 4.0. Now, the feature is also available for non-localisable attributes when there is only one locale activated. This means that if you have a catalog with only one activated locale, you can now benefit from spell-check on all your text and text area attributes, if your locale is one of the supported languages.

Reviewer: I would go for something way shorter like:
```suggestion
For those of you who have a catalog with only one activated locale, we have good news for you. You can now benefit from spell-check on all your text and text area attributes, if your locale is one of the supported languages.
```

Reviewer: Maybe we should give the list of the supported locales for spell check right here?

Reviewer 2: I like Nono's version. more impactful. 

```suggestion
For those of you who have a catalog with only one activated locale, we have good news. You can now benefit from spell-checking on all your text and text area attributes, if your locale is one of the supported languages.
```

::: more

> [Understand data quality](../articles/articles/understand-data-quality.html)

Reviewer: ```suggestion
[What is data quality?](../articles/articles/understand-data-quality.html)
```
